data/styles/clear/include/Subways.mapcss

```diff
@@ -1,540 +1,476 @@
 /* ~~~~ CONTENT OF SUBWAYS ICONS ~~~~~
-
- 3.2 Custom Subway Station
+ 1. Generic Subway Station
+ 2. Custom Subway Station (sorted by comment name)
 
 If adding a new city, make sure to add
 the bbox to MatchCity() in generator/osm2type.cpp
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+/* 1. Generic Subway and light_rail Station */
+
+/* reset style for generic station because railway-station inherited from icons.mapcss */
+node|z0-[railway=station][transport=subway],
+node|z0-[railway=station][station=light_rail],
+{icon-image: none; text: none;}
+
+node|z0-[railway=station][transport=subway]::int_name,
+node|z0-[railway=station][station=light_rail]::int_name,
+{icon-image: none; text: none;}
+/*TODO: for some reason, int_name rules and normal rules have to be
+separate, otherwise generating drules fails with a priority error */
+
+node|z13-[railway=station][transport=subway],
+{icon-image: subway-s.svg;icon-min-distance: 1;}
```

Comment:
While we are at it let's remove `icon-min-distance: 1;`. This tiny adjustment is useless anyway.